src/analytics/analytics-step-tracker.ts

```diff
@@ -137,7 +137,34 @@ export default class AnalyticsStepTracker implements StepTracker {
             coupons,
             extraItemsData,
             lineItems,
-        }));
+        });
+
+        if (features?.['DATA-6891.missing_orders_within_GA'] && this.analytics.hasPayloadLimit(payload)) {
+            this.analytics.hit('transaction', {
+                '&ti': payload.orderId,
+                '&ta': payload.affiliation,
+                '&tr': payload.revenue,
+                '&ts': payload.shipping,
+                '&tt': payload.tax,
+                '&tcc': payload.coupon,
+                '&cu': payload.currency,
+            });
+
+            payload.products.map(product => {
+                this.analytics.hit('item', {
```

Comment:
this object seems to be the same as the previous one, might be worth re-using it?

```js
const compressedPayload = {
                '&ti': payload.orderId,
                '&ta': payload.affiliation,
                '&tr': payload.revenue,
                '&ts': payload.shipping,
                '&tt': payload.tax,
                '&tcc': payload.coupon,
                '&cu': payload.currency,
            })